src/makeRobot.js

```diff
@@ -37,8 +37,61 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goLeft(number = 1) {
+      const value = number < 0 ? 0 : number;
+
+      this.coords.x -= value;
```

Comment:
```suggestion
      if (number > 0) {
        this.coords.x -= value;
      }
```
You can just change coord only if number is positive. 
Or you can use ternary operator directly to this.coords.x if you want